src/main/java/org/example/controllers/OrderController.java

```diff
@@ -0,0 +1,31 @@
+package org.example.controllers;
+
+import org.example.entities.Order;
+import org.example.repository.OrderRepository;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@ResponseBody
+public class OrderController {
+    private OrderRepository orderRepository;
+
+    public OrderController(OrderRepository orderRepository) {
+        this.orderRepository = orderRepository;
+    }
+
+    @RequestMapping("/{id}")
+    public Order getOrder(@PathVariable long id){
+        return orderRepository.getById(id);
+    }
+
+    @RequestMapping("/orders")

```

Comment:
revert to `@GetMapping`. I don't see any point, why did you replace it with `@RequestMapping`, as I wanted you to put `@RequestMapping` on the class level.